lib/growthbook/inline_experiment.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module Growthbook
+  class InlineExperiment
+    # @returns [String]
+    attr_accessor :key
+
+    # @returns [Any]
+    attr_accessor :variations
+
+    # @returns [Bool]
+    attr_accessor :active
+
+    # @returns [Integer, nil]
+    attr_accessor :force
+
+    # @returns [Array<Float>, nil]
+    attr_accessor :weights
+
+    # @returns [Float]
+    attr_accessor :coverage
+
+    # @returns [Hash, nil]
+    attr_accessor :condition
```

Comment:
lot of  writable attributes [invites client classes to become too intimate with its inner workings, and in particular with its representation of state](https://github.com/troessner/reek/blob/v6.0.6/docs/Attribute.md#introduction)